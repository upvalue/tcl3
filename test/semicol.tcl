puts 5; puts 6
puts 5;;; puts 6
