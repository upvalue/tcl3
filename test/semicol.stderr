{"type": "TK_ESC", "begin": 0, "end": 4, "body": "puts"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_ESC", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOL", "begin": 6, "end": 8, "body": "; "}
{"type": "TK_ESC", "begin": 8, "end": 12, "body": "puts"}
{"type": "TK_SEP", "begin": 12, "end": 13, "body": " "}
{"type": "TK_ESC", "begin": 13, "end": 14, "body": "6"}
{"type": "TK_EOL", "begin": 14, "end": 15, "body": "\n"}
{"type": "TK_ESC", "begin": 15, "end": 19, "body": "puts"}
{"type": "TK_SEP", "begin": 19, "end": 20, "body": " "}
{"type": "TK_ESC", "begin": 20, "end": 21, "body": "5"}
{"type": "TK_EOL", "begin": 21, "end": 25, "body": ";;; "}
{"type": "TK_ESC", "begin": 25, "end": 29, "body": "puts"}
{"type": "TK_SEP", "begin": 29, "end": 30, "body": " "}
{"type": "TK_ESC", "begin": 30, "end": 31, "body": "6"}
{"type": "TK_EOL", "begin": 31, "end": 32, "body": "\n"}
{"type": "TK_EOF", "begin": 31, "end": 32, "body": "\n"}
